airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json

```diff
@@ -103,6 +103,19 @@
       "sync_mode": "incremental",
       "destination_sync_mode": "append",
       "cursor_field": ["updated_at"]
+    },
+    {
+      "stream": {
+        "name": "event",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": ["occurred_at"],
+        "source_defined_primary_key": [["id"]]
+      },
+      "sync_mode": "incremental",
+      "destination_sync_mode": "append",
+      "cursor_field": ["occurred_at"]
```

Comment:
I've no idea if this is necessary. Is this part used during integration test? Or is it just for documentation purposes? I guess we'll see when tests run 🤓